.github/workflows/ci.yml

```
@@ -0,0 +1,38 @@
+name: CI
```

今後もci が増えてくるので命名を lint にしたいです
```suggestion
name: Lint
```